providers/exoscale/exoscaleProvider.go

```diff
@@ -100,11 +103,23 @@ func (c *exoscaleProvider) GetDomainCorrections(dc *models.DomainConfig) ([]*mod
 			rec.SetTarget(r.Content)
 		case "MX":
 			if err := rec.SetTargetMX(uint16(r.Prio), r.Content); err != nil {
-				panic(fmt.Errorf("unparsable record received from dnsimple: %w", err))
+				panic(fmt.Errorf("unparsable record received from exoscale: %w", err))
+			}
+		case "SRV":
+			var err error
+			parts := strings.Fields(r.Content)
+			if len(parts) == 3 {
```

Comment:
Please add a comment explaining why there are code paths for  `len == 3` and `len != 3`.   (I'm unclear why both are needed. Is this for handling empty targets?)

Reply:
yeah, this is for handling empty targets — I modeled it after the `dnsimple` provider

Reply:
testing a simplified version now, will add a comment after

Reply:
ended up removing the extra code path and just skipping the empty target integration test.